stable/katafygio/Chart.yaml

```
@@ -5,7 +5,7 @@ name: katafygio
 home: https://github.com/bpineau/katafygio
 sources:
 - https://github.com/bpineau/katafygio
-version: 1.0.0
+version: 1.0.1
```

bump to 
```suggestion
version: 1.1.0
```

adding new feature not a bug fix